### PR TITLE
Validate Model Texture sizes

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Every model texture image must be the same size in order for the volume to render properly. The colormap validation is okay because we just sample the first row of the image.

- closes issue #113

### CODE REVIEW

**Review by broarr — APPROVED** [review]
This all looks good tome!

File: src/components/AframeScene/AframeScene.jsx [hunk 1]
```diff
@@ -90,7 +94,8 @@ function AframeScene({
           <ul>
             {errors.map((e) => (
               <li key={e}>
-                {e.message}: &nbsp; {e.cause.message}
+                {/* {e.message}: &nbsp; {e.cause.message} */}
```
[the remark this answers is not in the document]
Comment (RobertGemmaJr) [reply src/components/AframeScene/AframeScene.jsx]: Done (:

**Comment by RobertGemmaJr** [comment]
I added a `console.error` for the errors that get bubbled up into aframe. Figured why not?

**Review by broarr — APPROVED** [review]

**Comment by broarr** [comment]
I'm not mad about a well placed `console.error`!

**Pull request merged**